.github/workflows/deploy.yml

```diff
@@ -0,0 +1,32 @@
+name: Deployment
+
+on:
+  release:
+    types: [created]
+    branches: [ master ]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Prepare credentials
+        env:
+          MAVEN_PUBRING_GPG_BASE64: ${{ secrets.MAVEN_PUBRING_GPG_BASE64 }}
+          MAVEN_SECRING_GPG_BASE64: ${{ secrets.MAVEN_SECRING_GPG_BASE64 }}
+        run: echo $MAVEN_PUBRING_GPG_BASE64 > pubring.gpg && echo $MAVEN_SECRING_GPG_BASE64 > secring.gpg
+      - name: Deploy on Sonatype
+        env:
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          MAVEN_SECRING_PASSWORD: ${{ secrets.MAVEN_SECRING_PASSWORD }}
+        run: mvn deploy -P release --settings .github/mvn_settings_deploy.xml -DskipTests
+      - name: Clean credentials
+        if: always()
```

Comment:
Is this line necessary?

Reply:
Could you specify which line you mean?

Reply:
if: always() Is needed to remove the secrets files even if a previous step fails

Reply:
The comment always refers to the line right above. Thanks for the piece of info